### PR TITLE
Add SUDO to installations, to fix broken image

### PR DESCRIPTION
**Pull request by rsletta**
I found out that starting a Minecraft server from your image, resulted in a crash. Upon investigation, I found out that SUDO was missing, and that broke the installation. I added SUDO to the installation in the first RUN step, making the installation successful.

### CODE REVIEW

**Comment by rlenferink** [comment]
Hi @rsletta, many thanks for your input. I deed some testing and indeed it seems the `sudo` binary was missing. However, I think it isn't necessary for now, so removing the `sudo` prefix could be a solution as well.

If you want, can you update this PR so `sudo` is not used anymore (see my suggestions)?

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ FROM   ubuntu:18.04
 ENV    DEBIAN_FRONTEND noninteractive
 
 # Download and install everything from the repos.
-RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common
+RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common sudo
```
Comment (rlenferink) [inline Dockerfile]: As mentioned, can you remove `sudo` here

```suggestion
RUN    apt-get --yes update; apt-get --yes upgrade; apt-get --yes install software-properties-common
```

File: Dockerfile [hunk 2]
```diff
@@ -12,7 +12,7 @@ FROM   ubuntu:18.04
...
 RUN    sudo apt-add-repository --yes ppa:webupd8team/java; apt-get --yes update
```
Comment (rlenferink) [inline Dockerfile]: As mentioned, `sudo` should be removed from here as well

```suggestion
RUN    apt-add-repository --yes ppa:webupd8team/java; apt-get --yes update
```

File: Dockerfile (hunk 1 above)
Comment (rsletta) [reply Dockerfile]: I managed to screw this up, so I closed the PR. Will fix it locally, and submit a new PR. :)

Comment (rlenferink) [reply Dockerfile]: No problem, I'm happy to review your next PR :)

**Pull request closed without merging**